CONTRIBUTING.md

```diff
@@ -163,34 +154,50 @@ Some tips:
 
 [ocamlformat]: https://github.com/ocaml-ppx/ocamlformat
 
-## VS Code Extension guide
-
-The VS Code extension is setup as a standard `npm` type-script + React package
-using `esbuild` as the bundler. The extension has a main component in
-`editor/code/src/` and some webviews components written using React under
-`editor/code/views`.
-
-There are two ways to work with the VS Code extension: you can let VS Code
-itself take care of building it (preferred setup), or you can build it manually.
-
-First, run `npm install` in `editor/code`:
+## Client guide (VS Code Extension)
 
-```sh
-(cd editor/code && npm i)
-```
+The VS Code extension is setup as a standard `npm` Typescript + React package
+using `esbuild` as the bundler. 
 
-That will setup the required packages as it is usual. You can run `package.json`
-scripts the usual way:
+### Setup 
+1. Navigate to the editor folder
+    ```sh
+    cd editor/code
+    ```
+1. Install `esbuild`:
+    ```sh
+    npm i esbuild
+    ```
```

Comment:
Yup, I wasn't sure whether to combine both or not since the other seemed to be part of the "manual" approach. I actually just did `npm i` when following the instructions (seems simpler to me).